TEI/ParlaMint-schemaSpecs.editing.odd.xml

```diff
@@ -1,1703 +1,4094 @@
+                  <affiliation from="1979-05-03" ref="#parla.lower" role="MP" to="1983-06-09"/>
+                  <affiliation from="1979-05-11" ref="#PoGB" role="minister" to="1981-01-05"/>
+                  <affiliation from="1981-01-05" ref="#PoGB" role="minister" to="1981-09-14"/>
+                  <affiliation from="1981-09-14" ref="#PoGB" role="minister" to="1987-06-13"/>
+                  <affiliation from="1983-06-09" ref="#parla.lower" role="MP" to="1987-06-11"/>
+                  <affiliation from="1987-06-11" ref="#parla.lower" role="MP" to="1992-04-09"/>
+                  <affiliation from="1987-06-13" ref="#PoGB" role="minister" to="1990-01-02"/>
+                  <affiliation from="1992-04-09" ref="#parla.lower" role="MP" to="1997-05-01"/>
+                  <affiliation from="1997-05-01" ref="#parla.lower" role="MP" to="2001-06-07"/>
+                  <affiliation from="2001-07-03" ref="#parla.lower" role="MP"/>
+                  <affiliation from="2001-07-03"
+                               ref="#party.CON"
+                               role="member"
+                               to="2016-09-01"/>
+                  <affiliation from="2016-09-01" ref="#party.I" role="member"/>
+                  <affiliation from="2016-09-01" ref="#parla.upper" role="speaker"/>
+               </person>
+         </egXML>
+      </exemplum>
+       <exemplum xmlns:xi="http://www.w3.org/2001/XInclude" xml:lang="en">
+         <egXML xmlns="http://www.tei-c.org/ns/Examples" source="#ParlaMint">
+	   <p>The affiliation element can also include an ana attribute, which points to the appropriate legislative period when the person was affiliated with the specified organisation.</p>
+	  <person xml:id="BahŽibertAnja">
+	    <persName>
+              <surname>Bah</surname>
+              <surname>Žibert</surname>
+                     <forename>Anja</forename>
+            </persName>
+                       <affiliation role="MP"
+                         ref="#DZ"
+                         from="2014-08-01"
+                         to="2018-06-21"
+                         ana="#DZ.7"/>
+            <affiliation role="member"
+                         ref="#party.SDS.2"
+                         from="2014-08-01"
+                         to="2018-06-21"
+                         ana="#DZ.7"/>
+            <affiliation role="MP" ref="#DZ" from="2018-06-22" ana="#DZ.8"/>
+            <affiliation role="member" ref="#party.SDS.2" from="2018-06-22" ana="#DZ.8"/>
+          </person>
+	</egXML>
+       </exemplum>
+      <remarks xml:lang="en">
+         <p>Affiliation to organization can be done in three ways (sorted from most concrete - the best solution):
+      <list>
+               <item>
+                  <att>ref</att>
+               </item>
+               <item>
+                  <att>key</att>
+               </item>
+               <item>text value</item>
+            </list>
+         </p>
+      </remarks>
```

Comment:
```suggestion
```

We want only `ref` to be used for affiliations, discussed here: #177

Reply:
Agreed, committed.